OpenRA.Mods.Common/Scripting/Global/UtilsGlobal.cs

```diff
@@ -85,6 +85,20 @@ public LuaTable Skip(LuaTable table, int numElements)
 			return t;
 		}
 
+		[Desc("Concatenates two Lua tables into a single table.")]
+		public LuaTable Concat(LuaValue[] firstCollection, LuaValue[] secondCollection)
+		{
+			var t = Context.CreateTable();
+
+			foreach (var e in firstCollection)
+				t.Add(t.Count + 1, e);
```

Comment:
I see a CopyTo function in the LuaTable class. Could that work better here?

Reply:
I don't see a fitting overload nor that that would be necessary.